CHANGELOG

```diff
@@ -75,6 +75,7 @@
 * Braintree - Update transaction hash to include processor_authorization_code [yunnydang] #4718
 * CyberSourceRest: Add apple pay, google pay [gasb150] #4708
 * CybersourceREST - Void | Verify [sinourain] #4695
+* CyberSourceRest: Add ACH Support [edgarv09] #4722
```

Comment:
Thanks - I was about to say this is done by the merging Spreedly engineer, but honestly there's no reason you all can't add it yourself. We just need to deal with any reordering due to conflict at time of merge 👍